docs/Clearwater_SNMP_Statistics.md

```diff
@@ -52,6 +52,14 @@ Sprout nodes provide the following statistics:
 * The average latency, variance, highest latency and lowest latency for requests to Homestead's `/impu/<public ID>/reg-data` endpoint, indexed by time period.
 * The average latency, variance, highest latency and lowest latency for requests to Homestead's `/impu/<public ID>/location` endpoint, indexed by time period.
 * The average latency, variance, highest latency and lowest latency for requests to Homer, indexed by time period.
+* The number of attempts, successes and failures for initial registrations, indexed by time period.
+* The number of attempts, successes and failures for re-registrations, indexed by time period.
+* The number of attempts, successes and failures for de-registrations, indexed by time period.
+* The number of attempts, successes and failures for third-party initial registrations, indexed by time period.
+* The number of attempts, successes and failures for third-party re-registrations, indexed by time period.
+* The number of attempts, successes and failures for third-party de-registrations, indexed by time period.
+* The number of attempts, successes and failures for AKA authentications, indexed by time period (AKA authentication attempts with a correct response but that fail due to the sequence number in the nonce being out of sync are counted as successes).
```

Comment:
Can this specify that it only covers registrations? (We added authentication of non-REGISTER requests this sprint, but those will be tracked by separate stats.)

This probably also applies to the MIB file descriptions.